server/controllers/entities/lib/properties/properties_values_constraints.js

```diff
@@ -37,6 +37,8 @@ module.exports = {
   'wdt:P31': extend(bases.uniqueEntity, { adminUpdateOnly: true }),
   // author
   'wdt:P50': extend(bases.humanEntity, { hasPlaceholders: true }),
+  // editor
+  'wdt:P98': bases.humanEntity,
```

Comment:
It seems Wikiproject:Books recommend a broader definition of P98 values than only humans. If you take a look at the P98 [example for a work](https://www.wikidata.org/wiki/Wikidata:WikiProject_Books) (as nothing of the matter is recommended for collections) the example shows the statement `Q25666 P98 Q299015`.

[15% of entities which belong to a P98 are not Q5s](https://query.wikidata.org/#select%20%3Feditor%20%3FeditorLabel%20%3Fcollection%20%3FcollectionLabel%20%7B%0A%20%20%3Fcollection%20wdt%3AP98%20%3Feditor.%0A%20%20filter%20not%20exists%20%7B%3Feditor%20wdt%3AP31%20wd%3AQ5%7D%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D) (3513/23794) and I believe a best effort cannot avoid 15% (and you may remember this sentence future self, its written to eternity now) 

[Among those, many looks like organisations](https://query.wikidata.org/#select%20%3FinstanceLabel%20%3Finstance%20%28count%28%3Finstance%29As%20%3Fcount%29%20%7B%0A%20%20%3Fcollection%20wdt%3AP98%20%3Feditor.%0A%20%20filter%20not%20exists%20%7B%3Feditor%20wdt%3AP31%20wd%3AQ5%7D%0A%20%20%3Feditor%20wdt%3AP31%20%3Finstance%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D%0Agroup%20by%20%3Finstance%20%3FinstanceLabel%0Aorder%20by%20desc%28%3Fcount%29). 

For example, can we really avoid editors such as Q202660 (Collège de France) ?

Since adding P98 to inventaire is a creation, i think it could be nice to also tackle this " `organisations` as full fledged citizen in inventaire data model", as partly discussed in #295 but for other props.

Reply:
@jum-s I agree on the principle, this let us with 4 options:
- let the current implementation, which is too restrictive: could be an easy first step
- remove restriction and accept any type of entity until we can more finely restrict input: would be equally trivial
- create an organization type and implement the possibility to have properties bound to several types (in this case humans+organizations): while that's what we should probably do on the long term, I would oppose picking this option as a premise to merging this PR as that's a guaranteed staled branch imho
- remove all references to P98 from this PR: trivial

Reply:
ouch 3 months to answer this... option 2 seems attractive, we need to consider what to do with values which would be created while implementing options 3 and which would not fit `humans` or `collectivity` (yes im planting seed for the naming)

Reply:
as `collection P98 publisher` is hardly used on Wikidata and because [wd community](https://www.wikidata.org/wiki/Property_talk:P98) tends to define P98 as a property dedicated to academic or scientific newspapers, it appeared that `P98` integration in an entity edit form can be on hold for the moment as it might add some confusion between P98 and P123, until a consensus for its usage will be found.